src/tools/dev/scripts/bv_support/bv_cmake.sh

```diff
@@ -427,7 +186,7 @@ function build_cmake
     elif [[ "$OPSYS" == "Linux" && "$C_COMPILER" == "xlc" ]]; then
         env CXX=xlC CC=xlc CXXFLAGS="" CFLAGS="" ./bootstrap --prefix="$VISITDIR/cmake/${CMAKE_VERSION}/$VISITARCH"
     else
-        env CC=${C_COMPILER} CXX=${CXX_COMPILER} CXXFLAGS="" CFLAGS="" ./bootstrap --prefix="$VISITDIR/cmake/${CMAKE_VERSION}/$VISITARCH"
+        env CC=${C_COMPILER} CXX=${CXX_COMPILER} CXXFLAGS="" CFLAGS="" ./bootstrap --prefix="$VISITDIR/cmake/${CMAKE_VERSION}/$VISITARCH" --parallel=8 ${CMAKE_BOOTSTRAP_FLAGS}
```

Comment:
should --parallel use value from build_visit's option?

Reply:
So, I've looked. We don't really have that number as a variable. AFAICT, all we have is `$MAKE_OPT_FLAGS` so if someone specified `-j8`, we get 8-way parallel makes with `$MAKE $MAKE_OPT_FLAGS`.

I would need disect the `$MAKE_OPT_FLAGS` variable, looking for `-j.[\d]+` to break that out that number. I can, if you think its worth it.

Reply:
@cyrush ok, I've tested this logic on linux and macos...

```
nprocs=$(echo $MAKE_OPT_FLAGS | tr ' ' '\n' | grep -A 1 -e -j | grep . | tr -d '\n' | tr -d 'j-')
```
And it works if `MAKE_OPT_FLAGS="-k -i -j 8' or `MAKE_OPT_FLAGS=-j8`

Should I do that?

Reply:
cmake's configure might actually support -j. 
I know cmake --build does support it

Reply:
If not -- I would just leave out the parallel vs more complex logic.

Reply:
Not seeing that. This is `./bootstrap` too (I dunno if that is what you mean by CMake's `configure`?)

FYI...

```
[scratlantis:build-mb-3.4.1-darwin-21-x86_64-release/thirdparty_shared/cmake-3.24.3] miller86% ./bootstrap --help

Usage: ./bootstrap [<options>...] [-- <cmake-options>...]
Options: [defaults in brackets after descriptions]
Configuration:
  --help                  print this message
  --version               only print version information
  --verbose               display more information
  --parallel=n            bootstrap cmake in parallel, where n is
                          number of nodes [1]
  --generator=<generator> generator to use (MSYS Makefiles, Unix Makefiles,
                          or Ninja)
  --enable-ccache         Enable ccache when building cmake
  --init=FILE             load FILE as script to populate cache
  --system-libs           use all system-installed third-party libraries
                          (for use only by package maintainers)
  --no-system-libs        use all cmake-provided third-party libraries
                          (default)
  --system-curl           use system-installed curl library
  --no-system-curl        use cmake-provided curl library (default)
  --system-expat          use system-installed expat library
  --no-system-expat       use cmake-provided expat library (default)
  --system-jsoncpp        use system-installed jsoncpp library
  --no-system-jsoncpp     use cmake-provided jsoncpp library (default)
  --system-zlib           use system-installed zlib library
  --no-system-zlib        use cmake-provided zlib library (default)
  --system-bzip2          use system-installed bzip2 library
  --no-system-bzip2       use cmake-provided bzip2 library (default)
  --system-liblzma        use system-installed liblzma library
  --no-system-liblzma     use cmake-provided liblzma library (default)
  --system-nghttp2        use system-installed nghttp2 library
  --no-system-nghttp2     use cmake-provided nghttp2 library (default)
  --system-zstd           use system-installed zstd library
  --no-system-zstd        use cmake-provided zstd library (default)
  --system-libarchive     use system-installed libarchive library
  --no-system-libarchive  use cmake-provided libarchive library (default)
  --system-librhash       use system-installed librhash library
  --no-system-librhash    use cmake-provided librhash library (default)
  --system-libuv          use system-installed libuv library
  --no-system-libuv       use cmake-provided libuv library (default)

  --bootstrap-system-libuv use system-installed libuv library for bootstrap
  --bootstrap-system-jsoncpp use system-installed jsoncpp library for bootstrap
  --bootstrap-system-librhash use system-installed librhash library for bootstrap

  --qt-gui                build the Qt-based GUI (requires Qt >= 4.2)
  --no-qt-gui             do not build the Qt-based GUI (default)
  --qt-qmake=<qmake>      use <qmake> as the qmake executable to find Qt

  --sphinx-info           build Info manual with Sphinx
  --sphinx-man            build man pages with Sphinx
  --sphinx-html           build html help with Sphinx
  --sphinx-qthelp         build qch help with Sphinx
  --sphinx-latexpdf       build PDF with Sphinx using LaTeX
  --sphinx-build=<sb>     use <sb> as the sphinx-build executable
  --sphinx-flags=<flags>  pass <flags> to sphinx-build executable

Directory and file names:
  --prefix=PREFIX         install files in tree rooted at PREFIX
                          [/usr/local]
  --bindir=DIR            install binaries in PREFIX/DIR
                          [bin]
  --datadir=DIR           install data files in PREFIX/DIR
                          [share/cmake-3.24]
  --docdir=DIR            install documentation files in PREFIX/DIR
                          [doc/cmake-3.24]
  --mandir=DIR            install man pages files in PREFIX/DIR/manN
                          [man]
  --xdgdatadir=DIR        install XDG specific files in PREFIX/DIR
                          [share]
```

Reply:
> If not -- I would just leave out the parallel vs more complex logic.

On crossroads, it was taking quite a bit of time to get through this so parallelizing really helped...given how many times I was re-running this.

Reply:
Ok, I removed it.